_includes/navigation.html

```diff
@@ -4,70 +4,70 @@
     {% if site.about_enabled == true %}
       <li>
         <a href="{{ 'about' | relative_url }}" title="About me">
-          <span class="icon icon-about"></span>
+          <svg class="icon-about"><use xlink:href="{% asset_path about.svg %}#icon-about"></use></svg>
```

Comment:
Wasn't expecting this to be such a PITA to implement. Have tested a whole bunch of scenarios now and confirm that the following implementations DO work, while the rest appear to be implemented inconsistently across browsers:

1. The current implemented method works as is just using xlink:href
2. Using href followed by xlink:href in that order works
3. Inline SVG's work

I don't have IE available to test further so I cannot confirm the above 3 scenarios work equally well for that browser. My preference is to go with option 2. At worst case, if the proposed SVG2 standard isn't adopted, there's a bit of extra bloat in the HTML that will have negligible affect.